gcp/appengine/frontend_handlers.py

```diff
@@ -515,6 +517,16 @@ def group_versions(versions):
   return groups
 
 
+def sort_version(versions):
```

Comment:
Can you add a type annotation to this function e.g. `(versions: list[string]) -> list[string]`